packages/thirdweb/biome.json

```diff
@@ -6,7 +6,9 @@
       "src/crypto/aes/lib/md5.ts",
       "src/utils/promise/p-limit.ts",
       "src/utils/bytecode/cbor-decode.ts",
-      "src/**/__generated__/**"
+      "src/**/__generated__/**",
+      "src/merkletree/MerkleTree.ts",
```

Comment:
i see u gave up

Reply:
mainly a bunch of 're-assigned params', could maybe just drop that rule instead